### PR TITLE
Add other python versions to .travis.yml

### PR DESCRIPTION
**Pull request by JayDwee**
Added 3.7, waiting on pysqlcipher 1.0.4 on PyPi to be released for python 3.9

### CODE REVIEW

**Comment by JayDwee** [comment]
Wailing on https://github.com/rigglemania/pysqlcipher3/pull/23 before adding python 3.9 support

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
 
-    python:
-    - "3.8"
+install:
+  - python --version
+  - sudo apt-get install -y sqlcipher
+  - sudo apt-get install -y libsqlcipher-dev
+  - pip3 install -r requirements.txt
+  - pip3 install dpytest==0.0.20
+  - pip3 install discord.py==1.3.4
```
Comment (VirajShah18) [inline .travis.yml:17]: Custom versions of dpytest and discord.py should no longer be needed from #149

**Review by VirajShah18 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (JayDwee) [reply .travis.yml:17]: Just need one of them to be merged to master, #149 was merged into #134, not master

**Comment by JayDwee** [comment]
pysqlcipher3 has been updated on git, just waiting for PyPi update to 1.0.4, then testing with python 3.9 should be possible

**Comment by JayDwee** [comment]
python 3.9 support added, although I would avoid using it on our servers until #106 is complete

**Review by VirajShah18 — APPROVED** [review]

**Pull request merged**